docs/libcudacxx/extended_api/math/round_down.rst

```diff
@@ -0,0 +1,35 @@
+.. _libcudacxx-extended-api-math-round-down:
+
+``round_down`` Round to the previous multiple
+=============================================
+
+.. code:: cuda
+
+   template <typename T, typename = U>
+   [[nodiscard]] __host__ __device__ inline
+   constexpr cuda::std::common_type_t<T, U> round_down(T a, U b) noexcept;
+
```

Comment:
Maybe a description of the parameters would also help. On a similar note, providing more descriptive names for the parameters could also help with an initial understanding. Again, not a hard suggestion. Sth like:

```
template <typename T, typename = U>
   [[nodiscard]] __host__ __device__ inline
   constexpr cuda::std::common_type_t<T, U> round_down(T value, U base_multiple) noexcept;

``value``: The value to be rounded down.
``base_multiple``:  The base multiple to which the value rounds down.
```
I suspect the descriptions and parameter names are conforming to the BLAS documentation, in that case disregard my concern. If we go with it, we should also apply it to `round_up` and `ceil_div` though.